### PR TITLE
Add missing split -> for-each migration

### PR DESCRIPTION
**Pull request by kdutkowski**

### CODE REVIEW

**Review by arkadius — APPROVED** [review]
LGTM

File: docs/MigrationGuide.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ To see the biggest differences please consult the [changelog](Changelog.md).
       * for base component (like `filter` or `split`): `NodeComponentInfo.forBaseNode("nodeId", ComponentType.Filter)`
       * for other: `NodeComponentInfo("nodeId", ComponentInfo("kafka-avro-source", ComponentType.Source))`
   * The same migration has to be applied to `ExceptionHandler.handling()` method.
-
+* [#2824](https://github.com/TouK/nussknacker/pull/2824) 'ProcessSplitterMigration' available to migrate node name from 'split' to 'for-each' (see [#2781](https://github.com/TouK/nussknacker/pull/2781)) - please turn it on you are using 'split' component
```
Comment (arkadius) [inline docs/MigrationGuide.md:69]: "please turn it on you are using" - should be "if you are using"

File: engine/flink/components/base/src/test/scala/pl/touk/nussknacker/engine/flink/util/transformer/ForEachTransformerSpec.scala [hunk 2]
```diff
@@ -37,6 +42,20 @@ class ForEachTransformerSpec extends FunSuite with FlinkSpec with Matchers with
...
+    val model = modelData(List(TestRecord()), collectingListener)
+
+    val testProcess = aProcessWithForEachNode(elements = "{'one', 'other'}", resultExpression = s"#$forEachOutputVariableName + '_1'")
+
+    val compiledSources: NonEmptyList[PotentiallyStartPart] = getCompiledSources(model, testProcess)
+
+    val forEachOutputVariable = compiledSources.head.nextParts.find(_.id == forEachNodeId)
+      .get.validationContext.variables(forEachOutputVariableName)
+
+    forEachOutputVariable.asInstanceOf[typing.TypedClass].klass shouldBe classOf[String]
```
Comment (arkadius) [inline engine/flink/components/base/src/test/scala/pl/touk/nussknacker/engine/flink/util/transformer/ForEachTransformerSpec.scala:56]: ca be just `shouldBe Typed[String]` without extracting klass?

**Review by arkadius — APPROVED** [review]

File: engine/flink/components/base/src/test/scala/pl/touk/nussknacker/engine/flink/util/transformer/ForEachTransformerSpec.scala (hunk 2 above)
Comment (kdutkowski) [reply engine/flink/components/base/src/test/scala/pl/touk/nussknacker/engine/flink/util/transformer/ForEachTransformerSpec.scala:56]: sure, wasn't aware it's possible like that :D

**Pull request merged**